spec/components/schemas/templates/base.ts

```diff
@@ -46,6 +46,7 @@ const base: SchemaObject = {
           senderId: {
             title: 'Sender ID',
             type: 'string',
+            nullable: true,
```

Comment:
Another thing, is it suppose to be allowed to be null the senderId?

Reply:
No, is not! This property (nullable) can be removed.